src/RpcClient/Models/RpcNep5Transfers.cs

```diff
@@ -0,0 +1,81 @@
+using Neo.IO.Json;
+using Neo.SmartContract;
+using Neo.Wallets;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+namespace Neo.Network.RPC.Models
+{
+    public class RpcNep5Transfers
+    {
+        public UInt160 UserScriptHash { get; set; }
+
+        public List<RpcNep5Transfer> Sent { get; set; }
+
+        public List<RpcNep5Transfer> Received { get; set; }
+
+        public JObject ToJson()
+        {
+            JObject json = new JObject();
+            json["sent"] = Sent.Select(p => p.ToJson()).ToArray();
+            json["received"] = Received.Select(p => p.ToJson()).ToArray();
+            json["address"] = UserScriptHash.ToAddress();
+            return json;
+        }
+
+        public static RpcNep5Transfers FromJson(JObject json)
+        {
+            RpcNep5Transfers transfers = new RpcNep5Transfers
+            {
+                Sent = ((JArray)json["sent"]).Select(p => RpcNep5Transfer.FromJson(p)).ToList(),
+                Received = ((JArray)json["received"]).Select(p => RpcNep5Transfer.FromJson(p)).ToList(),
+                UserScriptHash = json["address"].AsString().ToScriptHash()
+            };
+            return transfers;
+        }
+    }
+
+    public class RpcNep5Transfer
+    {
+        public ulong TimestampMS { get; set; }
+
+        public UInt160 AssetHash { get; set; }
+
+        public UInt160 UserScriptHash { get; set; }
+
+        public BigInteger Amount { get; set; }
+
+        public uint BlockIndex { get; set; }
+
+        public ushort TransferNotifyIndex { get; set; }
+
+        public UInt256 TxHash { get; set; }
+
+        public JObject ToJson()
+        {
+            JObject json = new JObject();
+            json["timestamp"] = TimestampMS;
+            json["asset_hash"] = AssetHash.ToString();
```

Comment:
We should define a json coding style rules. Use '_'  or camelcase.

Reply:
Now, `snake_case` is more common in current JSON.  Camelcase seems only appear in  `entryPoint`, `returnType`, `tcpPort` and `vmState`.

Reply:
Maybe we should modify manifest to use `snake_case` but I think that we should unify the behaviour

Reply:
Yes, the unifying will change both `neo` and `neo-modules` code. Maybe we can raise an issue and have a discussion on using `snake_case` or `camelCase`.